### ISSUE TITLE
Add iRobot Roomba support

### ISSUE THREAD

**Issue by taugusti** [benct/lovelace-xiaomi-vacuum-card#24]
It would be fantastic if the card could also be used with the hass roomba component:
https://www.home-assistant.io/integrations/roomba/

**Comment by benct** [comment]
Sure! I need to know what kind of data and services that integration provides though Could you post the dev-info and/or screenshots like the ones provided in issue #23?

**Comment by taugusti** [comment]
I hope this is what you need:
<img width="425" alt="a3" src="https://user-images.githubusercontent.com/15052792/71720797-34099e00-2e23-11ea-902a-ec87219dbf24.PNG">
<img width="225" alt="a1" src="https://user-images.githubusercontent.com/15052792/71720799-34a23480-2e23-11ea-91bb-6826c21b659f.PNG">
<img width="300" alt="a2" src="https://user-images.githubusercontent.com/15052792/71720801-34a23480-2e23-11ea-88c0-99ac6a209318.PNG">

**Comment by benct** [comment]
Added in version `3.0.0`, try adding `vendor: roomba`. Let me know if something is wrong :)

**Comment by taugusti** [comment]
Hi @benct 
I have currently no contact to the roomba - for some strange reason it is rejecting connections...
I tried to install the component anyway - but I kept getting "component not found".
Eventually I removed the version information:
```
resources:
  - url: /local/xiaomi-vacuum-card.js
    type: module

```
And then it worked! After that I reapplied the version number - and now the component shows up!
Really strange and obviously nothing with your component to do...
- but maybe a solution others can use if you put it in the instructions?

In any case I will test as soon as my roomba gets back on the grid.

**Comment by taugusti** [comment]
Ok... after I upgraded hass to 103.6 the roomba got back online... strange...
My theory is that it is related to the iRobot app blocking hass.
There are notes that there are only one channel in the Roomba MQTT-server and if the app uses that it blocks HASS.
Not sure but it seems likely.

Only issue now is that the true / false texts are a bit strange:

![image](https://user-images.githubusercontent.com/15052792/72202953-a9015700-3465-11ea-8f04-a89236926cb0.png)

And... I would like to get rid of the background image. I have deleted the image in www/img but it still shows up. How do I get rid of the image?

My config:
```
                  - type: custom:xiaomi-vacuum-card
                    entity: vacuum.baby_5
                    name: Baby 5
                    vendor: roomba
```
